tests/semantic_analyser.cpp

```diff
@@ -979,6 +983,21 @@ TEST(semantic_analyser, watchpoint)
   test("watchpoint::0x0:8:rww { 1 }", 1);
 }
 
+TEST(semantic_analyser, kfunc)
+{
+  BTF btf;
+
+  if (!btf.has_function("memcpy"))
```

Comment:
can we fake the btf data so that we don't have to skip the tests?

Reply:
yep, will do